gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts

```diff
@@ -78,6 +78,33 @@ export default class ProposeOffchainConfig extends SolanaCommand {
   }
 
   static makeInputFromRDD = (rdd: any, stateAddress: string): OffchainConfig => {
+    if (!rdd) {
```

Comment:
We should expect that we always receive `rdd` correctly here. We should check for it before calling `makeInputFromRDD`